scripts/setup-windows.ps1

```diff
@@ -0,0 +1,32 @@
+[CmdletBinding()]
```

Comment:
I am not entirely for the idea of scripts for dependency installation.

Please rename this file to `dependencies-install.ps1`

Reply:
I wonder, where did you get the script's name from? 🤔